update drivedb.h (XITC SS6000SES3*)

weiyulong-xitc opened the pull request:
add xitc ssd

weiyulong-xitc reviewed (commented):
ok.

chrfranke commented:
Looks good so far. Please also do the following changes. Then the PR will be ready to merge:
- Improve readability by spaces behind ',' and indentation of the second comment line as done in the other entries:
  ```diff
  -  { "XITC SSDs",//tested with XITC SS6000SES327-7680G/F6600002,XITC SS6000SES3N6-240GB/F6600002,
  -  //XITC SS6000SES327-3840G/F6600002,XITC SS6000SES327-1920G/F6600002
  +  { "XITC SSDs",// tested with XITC SS6000SES327-7680G/F6600002, XITC SS6000SES3N6-240GB/F6600002,
  +      // XITC SS6000SES327-3840G/F6600002, XITC SS6000SES327-1920G/F6600002
       ...
  -    "","",
  +    "", "",
  ```
- Provide sample `smartctl -x -a` outputs for the other three testcases listed behind `tested with ...`. Otherwise these should not be listed.
- Further simplify the regex, for example:
  ```diff
  -    "XITC SS6000SES3(27|N6)-(240|480|960|1920|3840|7680)(G|GB)",
  +    "XITC SS6000SES3(27|N6)-(24|48|96|192|384|768)0GB?",
  ```

weiyulong-xitc commented:
- Improve readability by spaces behind ',' and indentation of the second comment line as done in the other entries:
   Yep, that looks much cleaner now. Fixed!
- Further simplify the regex,
   Perfect, that's much cleaner. I've made the change.
- Added missing test items
  [XITC SS6000SES3N6-240GB.log](https://github.com/user-attachments/files/21909464/XITC.SS6000SES3N6-240GB.log)
  [XITC SS6000SES327-1920G.log](https://github.com/user-attachments/files/21909465/XITC.SS6000SES327-1920G.log)
  [XITC SS6000SES327-3840G.log](https://github.com/user-attachments/files/21909466/XITC.SS6000SES327-3840G.log)
  [XITC SS6000SES327-7680G.txt](https://github.com/user-attachments/files/21909468/XITC.SS6000SES327-7680G.txt)
Please review again @chrfranke
